input/drivers/android_input.c

```diff
@@ -1398,6 +1395,13 @@ static void handle_hotplug(android_input_t *android,
    else if (strstr(android_app->current_ime, "com.hexad.bluezime"))
       strlcpy(name_buf, android_app->current_ime, sizeof(name_buf));
 
+   reserved_port = input_device_get_reserved_port(
+      vendorId, productId, device_name);
+
+   if (reserved_port > -1) {
```

Comment:
Can you get rid of the brackets here? For a single line there should be no brackets